About/about.html

```diff
@@ -0,0 +1,158 @@
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.3.0/css/font-awesome.css"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="../Sass/style.css" />
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+  </head>
+  <body>
+    <header>
+      <nav class="mobileNav d-md-none">
+        <i class="fa fa-bars pt-4 fa-3x" id="hamburger" aria-hidden="true"></i>
+      </nav>
+      <nav class="desktopNav mt-3">
+        <div
+          class="destopNavItems d-flex align-items-center justify-content-between"
+        >
+          <div class="logo ms-5 d-none d-md-block">
+            <a href="../index.html"
+              ><img src="../img/logo.jpeg" alt="logo"
+            /></a>
+          </div>
+          <div id="desktopList" class="d-none d-md-flex">
+            <img
+              id="closeX"
+              class="d-none ps-5 mb-5"
+              src="../img/close.png"
+              alt=""
+            />
+            <ul
+              class="items list-unstyled d-flex gap-4 align-items-center d-md-flex flex-column flex-md-row pt-5 pt-md-0"
+            >
+              <li class="ulLi">
+                <a class="text-decoration-none text-dark" href="#event"
+                  >Recent Events</a
+                >
+              </li>
+              <li class="ulLi">
+                <a
+                  class="text-decoration-none text-dark text-md-dark"
+                  href="#sponsors"
+                  >Sponsors</a
+                >
+              </li>
+              <li class="ulLi ms-5 me-5 border border-danger p-2 bg-danger">
+                <a class="text-decoration-none text-white" href="../index.html"
+                  >Home</a
+                >
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <h3 class="text-center">"Sharing with the world"</h3>
+      <h1 class="text-center col-md-6 mx-auto">
+        THE PERFECT BLUEPRINT TO E-COMMERCE 2022
+      </h1>
+    </header>
+
+    <main class="aboutMain col-md-12">
+      <div class="aboutHero hero">
+        <p class="text-center bg-white col-md-6 mx-auto p-3">
+          The eye-opening e-comm whitepaper brings together the community of
+          experts and academics who compromise the creative affiliate network in
+          different countries every two years. <br />
+          This year we hope to expand our invitation list including
+          organizations and individuals
+        </p>
+        <p class="text-center pb-5 pt-3 col-md-6 mx-auto">
+          Kindly contact us via email for any further question about OXJAYENT
+          E-COMM WHITEPAPER 2022 <br />
+          support@oxjayent.com
+        </p>
```

Comment:
- [ ] Kindly apply the `font-weight` suggested by the previous reviewer so that your design becomes identical to that of the design template chosen. I would advise you to use a `font-weight` with the value `bold` and change the `color` of the text to your `var(--bs-gray-dark)`. 👍

Reply:
font weight and bold tag applied